### PR TITLE
feat : create medical record

### PR DESCRIPTION
**Pull request by andrt2607**

### CODE REVIEW

File: internal/pkg/handler/medical_record.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+	//validate request input
+	if err := request.Validate(); err != nil {
+		errs.NewValidationError("input validation error", err).Send(ctx)
+		return
+	}
+
+	//get user role
+	userId := ctx.Value("userID").(string)
+	fmt.Print("ini usernip : ", userId)
```
Comment (hnfnfl) [inline internal/pkg/handler/medical_record.go]: Delete debug print

File: internal/pkg/service/medical_record.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	"fmt"
+	"halo-suster/internal/db/model"
+	"halo-suster/internal/pkg/dto"
+	"halo-suster/internal/pkg/errs"
+	"time"
+)
+
+func (s *Service) CreateMedicalRecord(medicalRecord model.MedicalRecord) errs.Response {
+	var err error
+	fmt.Println("sudah disini")
```
Comment (hnfnfl) [inline internal/pkg/service/medical_record.go]: Delete this too

**Review by hnfnfl — COMMENTED** [review]
I think better remove `validator *validator.Validate ` since its unused and we already use Ozzo validator

File: internal/pkg/service/medical_record.go (hunk 2 above)
Comment (andrt2607) [reply internal/pkg/service/medical_record.go]: done , u can check the new commit

**Review by hnfnfl — APPROVED** [review]

**Pull request merged**